### DISCUSSION TITLE
blog/2021-06-29-make-icc-plots-for-your-brms-irt-models/

### DISCUSSION THREAD

**Discussion by giscus** [ASKurz/blogdown5#25] [Comments]
# blog/2021-06-29-make-icc-plots-for-your-brms-irt-models/

The purpose of this blog post is to show how one might make ICC and IIC plots for **brms** IRT models using general-purpose data wrangling steps.

https://solomonkurz.netlify.app/blog/2021-06-29-make-icc-plots-for-your-brms-irt-models/

**Comment by lotitomaria** [comment]
Hi Solomon, thank you for this awesome resource. I'm trying to discern how I would change the code for the 1PL model if I have incorporated multiple random effects (for cross-classified data) into my model. Any advice?

**Reply by ASKurz** [reply]
Thanks for the kind words, @lotitomaria. It’s hard to convey this in the abstract, but yes I believe you can generalize parts of this workflow with a multilevel version of the model. However, I would use an `add_epred_draws()` based workflow, rather than an `as_draws_df()` based workflow. Part of the trick is thinking very carefully about the data statement you feed into `add_epred_draws()`, and how you average across the output. Sadly, I don’t think I have anything that looks like that on my blog.

**Reply by lotitomaria** [reply]
Awesome, I'll give it a try, thank you!
